ts/features/itwallet/presentation/proximity/screens/ItwProximityClaimsDisclosureScreen.tsx

```diff
@@ -44,19 +43,11 @@ export const ItwProximityClaimsDisclosureScreen = () => {
       // An undefined proximityDetails cannot be stored in the context, as any failure causes the machine to transition
       // to the Failure state.
       () => (
-        <LoadingScreenContent
-          contentTitle={I18n.t(
+        <ItwProximityLoadingScreen
```

Comment:
Isn't this the same as `LoadingScreenContent`?
If yes, do we really need a custom component like `ItwProximityLoadingScreen` just to handle one specific case?

Reply:
Addressed in https://github.com/pagopa/io-app/pull/7101/commits/635c11de134d3dcea0d4e84cf6412337830baa95 since it was redundant